src/ui/Pricing/Comparison/MoreFeaturesButton.svelte

```diff
@@ -5,7 +5,7 @@
 </script>
 
 <button
-  class="btn-0 fluid row hv-center"
+  class="btn-0 c-green fluid row hv-center"
```

Comment:
`[D-0-1]` `c-green` is redundant, since `btn-0` buttons inherit `accent` display and hover colors
https://github.com/santiment/san-webkit/blob/cba8c88455f7610d57fbcbee917f1ac724e57182/src/styles/buttons.css#L49-L51